lightning/src/chain/channelmonitor.rs

```diff
@@ -1568,6 +1568,7 @@ impl<Signer: Sign> ChannelMonitorImpl<Signer> {
 					L::Target: Logger,
 	{
 		for tx in self.get_latest_holder_commitment_txn(logger).iter() {
+			log_info!(logger, "Broadcasting local {}", log_tx!(tx));
```

Comment:
nit: could say "local commitment transaction" for clarity

Reply:
`log_tx` adds "commitment tx" for us.